src/raft.c

```diff
@@ -25,10 +25,10 @@ const char *RaftReqTypeStr[] = {
     "RR_APPENDENTRIES",
     "RR_REDISCOMMAND",
     "RR_INFO",
-    "RR_COMPACT",
+    "RR_DEBUG",
     "RR_CLIENT_DISCONNECT",
     "RR_SHARDGROUP_ADD",
-    "RR_SHARDGROUP_UPDATE",
+    "RR_SHARDGROUPS_REPLACE",
```

Comment:
my only nit is this.  yea, hard to keep in sync.  perhaps there's some gcc magic we can use to convert defines into strings so don't need this list?

Reply:
I don't know any simple magic for this and complex tricks isn't worth the effort most of the time. :) 

We can even delete this array just print integer enum values instead. This is only logging anyway. 

Alternatively, I added a commit. Manually, assigning enums to their strings and a test for if we have a gap in the array. Probably, this might help. Let me know which one you prefer.